crmsh/bootstrap.py

```diff
@@ -186,6 +195,61 @@ def init_sbd_manager(self):
         from .sbd import SBDManager
         self.sbd_manager = SBDManager(self)
 
+    def detect_platform(self):
+        """
+        Detect platform
+        """
+        self.is_s390 = "390" in os.uname().machine
+        if not self.is_s390:
+            self.cloud_type = utils.detect_cloud()
+
+    def determine_profile_type(self):
+        """
+        Determine profile type
+        """
+        if self.is_s390:
+            profile_type = self.S390_PROFILE_NAME
+        elif self.cloud_type:
+            profile_type = self.cloud_type
+        else:
+            return None
+        status("Detected \"{}\" platform".format(profile_type))
+        return profile_type
+
+    def load_specific_profile(self, profile_type):
+        """
+        Load specific profile
+        """
+        profile_dict = {}
+        if not profile_type:
+            return profile_dict
+
+        if profile_type in self.profiles_data:
+            status("Loading \"{}\" profile from {}".format(profile_type, self.PROFILES_FILE))
+            profile_dict = self.profiles_data[profile_type]
+        else:
+            status("\"{}\" profile is not exist in {}".format(profile_type, self.PROFILES_FILE))
```

Comment:
Change the msg by `profile does not exist`

Reply:
Changed, thank you!